e2e/node.go

```diff
@@ -42,3 +44,20 @@ func checkNodeHasLabel(c kubernetes.Interface, labelKey, labelValue string) erro
 	}
 	return nil
 }
+
+// List all nodes in the cluster (we have one), and return the IP-address.
+// Possibly need to add a selector, pick the node where a Pod is running?
```

Comment:
Yeah, with a pod selector with E2E pods, we can actually do that and it would be good as well

Reply:
that is an enhancement that may be needed later, it is not useful to add more complexity at this point

Reply:
Sure.. it can be a followup item